src/GovUk.Education.ExploreEducationStatistics.Content.ViewModels/AllMethodologiesViewModels.cs

```diff
@@ -1,35 +1,6 @@
 namespace GovUk.Education.ExploreEducationStatistics.Content.ViewModels;
 
-public class AllMethodologiesThemeViewModel : IComparable<AllMethodologiesThemeViewModel>
-{
-    public Guid Id { get; set; }
-
-    public string Title { get; set; } = string.Empty;
-
-    public List<AllMethodologiesTopicViewModel> Topics { get; set; } = new();
-
-    public void RemoveTopicNodesWithoutMethodologiesAndSort()
-    {
-        // Remove all publications without any methodologies
-        Topics.ForEach(topic => topic.RemovePublicationNodesWithoutMethodologiesAndSort());
-
-        // Remove all topics without any publications
-        Topics = Topics
-            .Where(topic => topic.Publications.Any())
-            .ToList();
-
-        Topics.Sort();
-    }
-
-    public int CompareTo(AllMethodologiesThemeViewModel? other)
-    {
-        if (ReferenceEquals(this, other)) return 0;
-        if (ReferenceEquals(null, other)) return 1;
-        return string.Compare(Title, other.Title, StringComparison.Ordinal);
-    }
-}
-
-public class AllMethodologiesTopicViewModel : IComparable<AllMethodologiesTopicViewModel>
+public class AllMethodologiesThemeViewModel : IComparable<AllMethodologiesPublicationViewModel>
```

Comment:
I think this line should have stayed the same? We're still comparing two `AllMethodologiesThemeViewModels`?